src/Conf.ml

```diff
@@ -210,27 +214,31 @@ end = struct
     let doc = generated_flag_doc ~allow_inline ~doc ~section in
     let docs = section_name section in
     let term =
-      Arg.(value & flag & info names_for_cmdline ~doc ~docs ~env)
+      Arg.(value & opt (some bool) None & info names_for_cmdline ~doc ~docs)
```

Comment:
You can keep the `flag` here and interpret `false` as `None` and `true` as `Some true`